frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultDetails.tsx

```diff
@@ -0,0 +1,147 @@
+import "./QuizResultDetails.scss";
+
+import { Card, CircularProgress, Paper, Typography } from "@material-ui/core";
+import React, { useState } from "react";
+
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import ErrorAlert from "../../../../../utils/Alerts/ErrorAlert";
+import EventIcon from "@material-ui/icons/Event";
+import PeopleIcon from "@material-ui/icons/People";
+import QuizResultChartsView from "./QuizResultViews/Charts/QuizResultChartsView";
+import QuizResultParticipantsView from "./QuizResultViews/Participants/QuizResultParticipantsView";
+import QuizResultQuestionsView from "./QuizResultViews/Questions/QuizResultQuestionsView";
+import TrendingUpIcon from "@material-ui/icons/TrendingUp";
+import { useQuizSummaryQuery } from "../../../../../../graphql";
+import { useTranslation } from "react-i18next";
+
+export interface QuizResultDetailsProps {
+  id: string;
+  eventName: string;
+}
+
+enum QuizResultView {
+  PARTICIPANTS,
+  QUESTIONS,
+  CHARTS,
+}
+
+export default function QuizResultDetails({
+  id,
+  eventName,
+}: QuizResultDetailsProps): React.ReactElement {
+  const { t } = useTranslation("common");
+  const { data, loading } = useQuizSummaryQuery({ variables: { id } });
+  const [activeView, setActiveView] = useState<QuizResultView>(
+    QuizResultView.PARTICIPANTS
+  );
+  const [raisedCard, setRaisedCard] = useState<QuizResultView | null>(null);
+
+  const onMouseOverChange = (viewType: QuizResultView) => {
+    setRaisedCard((prev) => (prev === viewType ? null : viewType));
```

Comment:
I think you can use the `activeView` here without any problems, it will be a little shorter (without arrow function) and more descriptive thatn `prev`

Reply:
`activeView` is for the currently chosen view and not the one that the mouse is over.

Reply:
Ok, makes sense.